doc/setup.md

```diff
@@ -79,7 +79,8 @@ have it in your PATH. This is used for javascript and interface testing.
 ### Running the entire suite once
 
 ```bash
-rake
+./bin/rake
```

Comment:
rake is so much easier to type! what about including `konacha:run` in default rake task so we can use it!?